library/aes.c

```diff
@@ -831,6 +835,9 @@ int mbedtls_aes_setkey_enc( mbedtls_aes_context *ctx, const unsigned char *key,
 #endif
     ) )
     {
+#if defined(MBEDTLS_VALIDATE_AES_KEYS)
+        ctx->crc = mbedtls_crc_update( 0, ctx->rk, keybits >> 3 );
```

Comment:
@armask , is it fine to use 0 as initial CRC value from security point of view? Or it is better to use non zero one ?